clang/test/CIR/CodeGen/builtins-memory.c

```diff
@@ -107,3 +110,41 @@ void test_memset_chk(void *dest, int ch, size_t n) {
   // CIR: cir.call @__memset_chk(%[[#DEST_LOAD]], %[[#CH_LOAD]], %[[#N_LOAD1]], %[[#N_LOAD2]])
   __builtin___memset_chk(dest, ch, n, n);
 }
+
+// TODO: The test should test intrinsic argument alignment, however, 
+// currently we lack support for argument attributes. 
+// Thus, added COM lines as a marker to remind us to change the test 
```

Comment:
What does `COM` means? Please use the more common approach, example: `FIXME: this should be <blah bleh bloh>`

Reply:
COM is a[ FileCheck directive](https://llvm.org/docs/CommandGuide/FileCheck.html#the-com-directive)
Added FIXME as maker;
use `COM: LLVM:` so when we are ready to fix the test later, we can simply use a script to do it automatically.

Reply:
wow, TIL, thanks!